### DISCUSSION TITLE
Compatibility with PyTorch 2.0

### DISCUSSION THREAD

**Discussion by nihpat19** [Project-MONAI/MONAI#6226] [Q&A] [answered]
Are any versions of MONAI compatible with PyTorch 2.0.0? If not, when can the release of a version that is compatible be expected?

**Comment by wyli** [comment, marked as answer]
Thanks for the question, the latest dev is tested with pytorch 2.0 for API compatibility, it'll be released in early May, for now you can try the release candidate `pip install --pre monai`.
